lib/common/options.js

```diff
@@ -132,6 +132,23 @@ Options.from = function (_options) {
     options.store = true
   }
 
+  // Set the name for global variables - should contain only letters
```

Comment:
I suggest moving this into a getter function inside the Builder module. This way, modules have a chance to override options. Also, we don't mutate options anymore.

Reply:
Thanks, I'll work something out.